### PR TITLE
Reduce code dupl

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

File: molecule/force_install/prepare.yml [hunk 1]
```diff
@@ -2,32 +2,9 @@
...
-        assets_server: "{{ lookup('env','MIDDLEWARE_DOWNLOAD_RELEASE_SERVER_URL') }}"
-
-    - name: "Download and deploy jws zips from {{ assets_server }}"
-      ansible.builtin.get_url:
-        url: "{{ asset }}"
-        dest: "/opt"
-        validate_certs: no
-      loop:
-        - "{{ assets_server }}/webserver/5.5.0/jws-5.5.0-application-server.zip"
-        - "{{ assets_server }}/webserver/5.6.0/jws-5.6.0-application-server.zip"
```
Comment (guidograzioli) [inline molecule/force_install/prepare.yml:25]: it is breaking here, this has two files

Comment (rpelisse) [reply molecule/force_install/prepare.yml:25]: Nice catch! I'll update that right away.

**Comment by rpelisse** [comment]
@guidograzioli if you don't have other comments... Merge?

**Review by guidograzioli — APPROVED** [review]
lgtm

**Pull request merged**